spec/deserializers/api/v2/deserializable_course_spec.rb

```diff
@@ -22,4 +22,15 @@
       expect(subject[:qualifications]).to eq('test')
     end
   end
+
+  describe "reverse_mapping" do
+    subject { described_class.new({}).reverse_mapping }
+
+    it "always contains all attributes" do
+      all_attributes = described_class.attr_blocks.keys
```

Comment:
This feels a bit too much like you're just repeating the implementation in the spec. One way to avoid it would be to:

1. Extract the attributes list in `DeserializableCourse` into a constant
2. Use the constant here.

That way you're avoiding having to maintain the list of attributes in multiple places.